docs/source/workflows/predictor_evaluation_workflows.rst

```diff
@@ -0,0 +1,264 @@
+Predictor Evaluation Workflows
+==============================
+
+A :class:`~citrine.informatics.workflows.predictor_evaluation_workflow.PredictorEvaluationWorkflow` evaluates the performance of a :doc:`Predictor <predictors>`.
+Each workflow is composed of one or more :class:`PredictorEvaluators <citrine.informatics.predictor_evaluator.PredictorEvaluator>`.
+
+Predictor evaluators
+--------------------
+
+A predictor evaluator defines a method to evaluate a predictor and any relevant configuration, e.g. k-fold cross-validation evaluation that specifies 3 folds.
+Minimally, each predictor evaluator specifies a name, a set of predictor responses to evaluate and a set of metrics to compute for each response.
+Evaluator names must be unique within a single workflow (more on that `below <#execution-and-results>`__).
+Responses are specified as a set of strings, where each string corresponds to a descriptor key of a predictor output.
+Metrics are specified as a set of :class:`PredictorEvaluationMetrics <citrine.informatics.predictor_evaluation_metrics.PredictorEvaluationMetric>`.
+When defining an evaluator, the top-level metrics should be the union of all metrics computed across all responses.
```

Comment:
I understand what you're saying here but it was confusing at first. I think that the information in line 16, that if a metric does not make sense for a response it will be ignored, needs to be understood before one can understand that the metrics argument takes a union of everything you are interested in.

Reply:
That's true. I'll switch the order and reword it a bit.